apps/prairielearn/elements/pl-code/pl-code.py

```diff
@@ -57,8 +57,10 @@ def __init__(self, **options: Any) -> None:
         pygments.lexer.Lexer.__init__(self, **options)
         self.compress = options.get("compress", "")
 
-    def get_tokens_unprocessed(self, text: str) -> list[tuple[int, _TokenType, str]]:
-        return [(0, Token.Text, text)]
+    def get_tokens_unprocessed(
+        self, text: str
+    ) -> Iterator[tuple[int, _TokenType, str]]:
+        return iter([(0, Token.Text, text)])
```

Comment:
This was the only functional change in this PR. I couldn't get PL running locally for some reason, so as part of the review, someone should fire up these changes locally to check that `pl-code` still works as expected (though I don't expect any issues).

Reply:
I checked `pl-code` locally, LGTM.